gradle-language-server/src/main/java/com/microsoft/gradle/semantictokens/TokenModifier.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+*******************************************************************************/
+package com.microsoft.gradle.semantictokens;
+
+import java.util.List;
+
+import org.eclipse.lsp4j.SemanticTokenModifiers;
+
+public enum TokenModifier {
+
+  DEFAULT_LIBRARY(SemanticTokenModifiers.DefaultLibrary);
+
+  private String genericName;
+  // See https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html
+  private static List<String> defaultLibrary = List.of("afterEvaluate", "allprojects", "ant", "apply", "artifacts",
```

Comment:
is it necessary to assign individual modifier for every closure type? How do you assign textmate scopes/styles in downstream themes then?